boards/ssrc/icicle/nuttx-config/bootloader/defconfig

```diff
@@ -54,9 +54,9 @@ CONFIG_MPFS_EMMCSD_MUX_EMMC=y
 CONFIG_MPFS_HART1_ENTRYPOINT=0x88000000
 CONFIG_MPFS_HART1_SBI=y
 CONFIG_MPFS_HART2_ENTRYPOINT=0xAFB00000
-CONFIG_MPFS_HART3_ENTRYPOINT=0x89000000
+CONFIG_MPFS_HART3_ENTRYPOINT=0x80200000
```

Comment:
Check these with Jani P. after the saluki memory map is settled, let's use the same

Reply:
Jani said that might change, but for now this is the correct value.